### ISSUE TITLE
Code formatting broken in  "Custom Tabs Implementation guide"

### ISSUE THREAD

**Issue by friedger** [google/WebFundamentals#8815]
**Page Affected:** https://developers.google.com/web/android/custom-tabs/implementation-guide

**What needs to be done?**
Code snippet in section https://developers.google.com/web/android/custom-tabs/implementation-guide#how_can_i_check_whether_the_android_device_has_a_browser_that_supports_custom_tab
is not completely shown in code formatting.

- [ ] Appropriate `type-something` label has been added.
- [ ] Appropriate `section-something` label has been added.

**Comment by cljazouli** [comment]
Good catch @friedger 👍 , I have a fix for that in https://github.com/google/WebFundamentals/pull/8826
